examples/argument_parser.py

```diff
@@ -1,3 +1,22 @@
+# Copyright (C) 2022. Huawei Technologies Co., Ltd. All rights reserved.
```

Comment:
Do we need headers for examples folder? Currently headers in examples are not checked by our CI.

Reply:
I had to add it when I moved this into the core. I'll remove it.